neptune/internal/abort.py

```diff
@@ -34,56 +37,36 @@ def __init__(self, pid):
 
     @staticmethod
     def requirements_installed():
-        # pylint:disable=unused-import,unused-variable,bad-option-value,import-outside-toplevel
-        try:
-            import psutil
-            return True
-        except ImportError:
-            _logger.warning('psutil is not installed. The experiment will not be abortable.')
```

Comment:
Nie widzę jak zmiany mają się do commit message'a. Z tego co rozumiem były wcześniej dwa różne komunikaty o braku psutil (dla aborta i dla metryk) i wciąż tak będzie, przy czym trochę lepszy kod.

Reply:
Yes, there will be 2, instead of 3 (right now we print the message about abort twice) :)
`requirements_installed` is called twice so I decided to move the error log to a place that should be executed onece per experiment.